src/app/demo/demo.component.html

```diff
@@ -1,51 +1,40 @@
 <div class="container">
-  <div class="nlsg-c-form-element__toggle-wrapper">
+  <div class="toggle-wrapper">
     <label>Freeze card</label>
-    <label class="nlsg-c-form-element__toggle-wrapper__toggle">
-      <input type="checkbox" (click)="open(content)" (change)="firstModel = !firstModel;">
-      <span class="nlsg-c-form-element__toggle-wrapper__toggle__slider round"></span>
+    <label class="toggle-wrapper__toggle">
+      <input type="checkbox" (click)="open(content)" (change)="openModel = !openModel;">
+      <span class="toggle-wrapper__toggle__slider round"></span>
     </label>
-
-    <label *ngIf="successFlag && !firstModel">
-      <img src="../assets/images/icons/Icon_Check_Green.svg"> Card Frozen Successfully </label>
-    <label *ngIf="unfreezeFlag && firstModel">
-      <img src="../assets/images/icons/Icon_Check_Green.svg"> Unfreeze Successfull </label>
-
+    <div *ngIf="displayLabel">
+      <label *ngIf="isCardFreeze && !openModel">
+        <span class="freezeCard"> </span>{{labels.freezeCard}}
+         </label>
+         <label *ngIf="!isCardFreeze && openModel">
+            <span class="freezeCard"></span> {{labels.unfreezeCard}} </label>
+    </div>
   </div>
 
-
-  <ng-template #content let-c="close" let-d="dismiss">
-
-    <div *ngIf="!firstModel">
-      <form class="nlsg-c-form-element">
-
+  <ng-template #content let-c="close" >
```

Comment:
again you have two set of codes of your modal window for freeze and unfreeze. I have asked you to write common code for both and based on the flag display the respective text

Reply:
used only one template and already removed firstModel in previous commit..